### DISCUSSION TITLE
Landsat tiles

### DISCUSSION THREAD

**Discussion by remi-braun** [CS-SI/eodag#782] [Show and tell]
Hello,

If needed, I created a shapefile containing centroids for Landsat tiles allowing to look for them by tiles, like for Sentinel-2.
The principle is exactly the same than for [this](https://eodag.readthedocs.io/en/stable/notebooks/tutos/tuto_search_location_tile.html) tutorial, with few minor changes:

```python
locations_yaml_content = """
shapefiles:
  - name: landsat_tiles
    path: {}
    attr: Name
""".format(
    "landsat_tiles.shp"
)
```
The tiles are formatted like this `\d{3}_\d{2}` (i.e. 200_30). They are coming from [this USGS](https://www.usgs.gov/media/files/landsat-wrs-2-scene-boundaries-kml-file) file. (note that USGS gives a centroid file but only for north latitude and west longitude).
This format is not exactly the same than on Landsat products (which is `\d{6}`, i.e. 200030)
This could be changed, but I wanted to stick as close as possible to the original format.

Note that works fine with Landsat as their tiles are always complete.
It can fail for Sentinel-2 with partial tiles (full of nodata, border of the swath) with the centroid falling outside their footprint.

Here is the file. You can of course add it in your tutorial if wanted 😄 
[landsat_tiles.zip](https://github.com/CS-SI/eodag/files/12238136/landsat_tiles.zip)

**Comment by sbrunato** [comment]
thank you very much @remi-braun !
